charts/topolvm/templates/controller/deployment.yaml

```diff
@@ -48,7 +48,7 @@ spec:
       {{- end }}
       containers:
         - name: topolvm-controller
-          image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
+          image: ghcr.io/syself/csi-topolvm:latest # "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
```

Comment:
this should be reverted and set in the values.yaml

Reply:
done.